--- assets.go
+++ assets.go
@@ -51,12 +51,6 @@
	for _, p := range publicAssets {
		b, err := fs.ReadFile(r.fsys, p)
		if err != nil {
-			if !r.liveMode {
-				// Only ignore if file is not found - maybe not embedded, but should usually be there
-				if !strings.Contains(err.Error(), "file does not exist") {
-					// We just continue on not found in case some test removes it
-				}
-			}
			continue
		}
