--- data_test.go
+++ data_test.go
@@ -75,6 +75,7 @@
		"showPages":      func() bool { return true },
		"loggedIn":       func() (bool, error) { return true, nil },
		"bookmarks":          func() (string, error) { return "", nil },
+		"bookmarksParserJS":  func() (template.JS, error) { return template.JS(""), nil },
		"bookmarkTabs": func() ([]TabInfo, error) {
			return []TabInfo{{Index: 0, Name: "", IndexName: "Main", Href: "/", EditHref: "/?edit=1", LastPageSha: ""}}, nil
		},
