--- funcs.go
+++ funcs.go
@@ -179,6 +179,13 @@
		"bookmarksExist": func() (bool, error) {
			return BookmarksExist(r)
		},
+		"bookmarksParserJS": func() (template.JS, error) {
+			b, err := templateFS.ReadFile("templates/bookmarks_parser.js")
+			if err != nil {
+				return "", err
+			}
+			return template.JS(string(b)), nil
+		},
		"bookmarksSHA": func() (string, error) {
			session := r.Context().Value(ContextValues("session")).(*sessions.Session)
			githubUser, _ := session.Values["GithubUser"].(*User)
